content/chronograf/v1.9/guides/create-alert-rules.md

```diff
@@ -22,141 +22,70 @@ Common alerting use cases that can be managed using Chronograf include:
 
-The purpose of this step is to generate enough load on your system to trigger an alert.
-More specifically, your idle CPU usage must dip below `80%`.
-On the machine that's running Telegraf, enter the following command in the terminal to start some `while` loops:
+1. Name the alert rule.
 
-```
-while true; do i=0; done
-```
+2. Choose from three alert types:
+    -  `Threshold` - alert if data crosses a boundary.
+    -  `Relative` - alert if data changes relative to data in a different time range.
+    -  `Deadman` - alert if InfluxDB receives no relevant data for a specified time duration.
 
-Let it run for a few seconds or minutes before terminating it.
-On most systems, kill the script by using `Ctrl+C`.
+3. Select the time series data. 
+    - Choose the time series data you want the Chronograf alert rule to use.
+    - Navigate through databases, measurements, tags, and fields to select all relevant data.
 
-### Step 2: View the alerts
+4. Define the rule condition(s).
+    - Condition options are determined by the alert type.  
 
-Go to the Slack channel that you specified in the previous section.
-In this example, it's the `#chronocats` channel.
+5. Select and configure the alert handler.
+    - The alert handler determines where the system sends the alert (the event handler).
+    - Chronograf supports several event handlers and each handler has unique configurable options.
+    - Multiple alert handlers can be added to send alerts to multiple endpoints.
 
-Assuming the first step was successful, `#ohnos` should reveal at least two alert messages:
+6. Configure the alert message.
+    - The alert message is the text that accompanies an alert.
+    - Alert messages are templates that have access to alert data.
+    - Available templates appear below the message text field.
+    - As you type your alert message, clicking the data templates will insert them at the end of whatever text has been entered.
 
-* The first alert message indicates that your idle CPU usage was `CRITICAL`, meaning it dipped below `80%`.
-* The second alert message indicates that your idle CPU usage returned to an `OK` level of `80%` or above.
+7. Click **Save Rule**.
 
-![See the alerts](/img/chronograf/1-6-alerts-slack-notifications.png)
+## View alert history
 
-You can also see alerts on the **Alert History** page available under **Alerting** in the left navigation.
+Chronograf lets you view your alert history on the **Alert History** page.
 
-![Chronograf alert history](/img/chronograf/1-6-alerts-history.png)
+To view a history of your alerts, click on
+**{{< icon "alert" "v2">}} Alerting** in the left navigation bar and select **Alert History**. 
+Do one of the following:
 
-That's it! You've successfully used Chronograf to configure an alert rule to monitor your idle CPU usage and send notifications to Slack.
+  - View a history of all triggered alerts.
+  - Filter alert history by type.  
+  - View alert history for a specified time range.  
```

Comment:
For the sake of thoroughness, let's add the following sections. I know they'll be short.

- Enable and disable an alert rule
- Delete an alert rule